application-services/custom/secrets/res/configuration.yaml

```
@@ -0,0 +1,27 @@
+Writable:
+  LogLevel: "INFO"
+  StoreAndForward:
+    Enabled: false
+  InsecureSecrets:
+    mqtt:
+      Path: "mqtt"
+      Secrets:
+        username: "mqtt-user"
+        password: "mqtt-pw"
+  Telemetry:
+    Interval: "0s"
+
+Service:
+  Host: "localhost"
+  Port: 59781 # Adjust if running multiple examples at the same time to avoid duplicate port conflicts
+  StartupMsg: "Secrets Example"
+
+MessageBus:
+  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are not enabled
+  Optional:
+    ClientId: "app-secrets"
+
+# Using default Trigger config from common config
```

This one uses HTTP Trigger

```suggestion
[Trigger]
Type="http"
```

---

Done